packages/calcite-components/src/components/carousel/carousel.tsx

```diff
@@ -300,6 +314,26 @@ export class Carousel
   //
   // --------------------------------------------------------------------------
 
+  private setMaxItemsToBreakpoint(width: number): void {
+    if (!breakpoints || !width) {
+      return;
+    }
+
+    const breakpointKeys = ["medium", "small", "xsmall", "xxsmall"];
```

Comment:
Could you get the keys from either of the breakpoints objects instead?

Reply:
Yerp, will do